README.md

```diff
@@ -64,11 +47,6 @@ of use:
 
     OPENSHIFT_AEROGEAR_PUSH_IP                         The IP address used to bind JBossAS
     OPENSHIFT_AEROGEAR_PUSH_HTTP_PORT                  The JBossAS listening port
-    OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY                  The token key for the SimplePush Server
-    OPENSHIFT_AEROGEAR_PUSH_CLUSTER_PORT               
```

Comment:
OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY was introduced for SimplePush.
OPENSHIFT_AEROGEAR_PUSH_CLUSTER_PORT isn't being used anymore so it can be removed (it used to be referenced in the jgroups subsystem configuration). The MESSAGING and REMOTING ports can probably also be removed from the README since they are not specific to the Push cartridge (they're related to WildFly).